build-system/tasks/runtime-test/index.js

```diff
@@ -97,18 +97,18 @@ function getConfig() {
     saucelabsBrowsers = argv.saucelabs ?
     // With --saucelabs, integration tests are run on this set of browsers.
       [
-        'SL_Chrome',
+        //'SL_Chrome',
         'SL_Firefox',
         // TODO(amp-infra): Restore this once tests are stable again.
         // 'SL_Safari_11',
-        'SL_Edge_17',
+        //'SL_Edge_17',
```

Comment:
@rsimha can you double check the list i disabled

Reply:
LGTM. Once we implement the sharding described in #20589, we can consider re-enabling these.